editor/src/components/editor/actions/actions.tsx

```diff
@@ -2030,6 +2033,9 @@ export const UPDATE_FNS = {
     }
   },
   UPDATE_EDITOR_MODE: (action: UpdateEditorMode, editor: EditorModel): EditorModel => {
+    if (!action.must && editor.multiplayer.following != null) {
```

Comment:
I have a negative gut feeling here, can't we decide whether to really update the editor mode where we dispatch the action?